src/components/bodyImageThumbnail.tsx

```diff
@@ -0,0 +1,35 @@
+// ----- Imports ----- //
+
+import React, { FC, ReactNode } from 'react';
+import { css } from '@emotion/core';
+import Image, { Props as ImageProps } from 'components/image';
+import { Pillar } from 'pillar';
+
+// ----- Setup ----- //
+
+const sizes = `(min-width: 140px)`;
```

Comment:
Does this also need a "source size value", as defined [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes)?

Is it possible this should be:

```ts
const sizes = '140px';
```